extensions/interactions/EndExploration/EndExploration.py

```diff
@@ -28,10 +28,12 @@ class EndExploration(base.BaseInteraction):
     name = 'End Conversation'
     description = (
         'Suggests recommendations for explorations to try next.')
```

Comment:
This might be a good time to update the description to specify that this interaction is also  used to finish the exploration, since it's no longer optional.